general/mem_manager.cpp

```diff
@@ -1301,6 +688,20 @@ bool MemoryManager::IsAlias_(const void *h_ptr)
    return maps->aliases.find(h_ptr) != maps->aliases.end();
 }
 
+bool MemoryManager::IsDanglingAlias_(const void *h_ptr)
+{
+   const auto alias_it = maps->aliases.find(h_ptr);
+   if (alias_it != maps->aliases.end())
+   {
+      const internal::Alias &alias = alias_it->second;
+      const void *base_h_ptr = static_cast<const char*>(h_ptr) + alias.offset;
```

Comment:
Just something I noticed here: I think the offset should subtracted from the alias address to get the base address.

Reply:
Thanks, fixed in d463bb916c6d6308144ab275717dc33ea6c49547.